tests/datastore_firestore/test_query.py

```diff
@@ -173,9 +173,9 @@ def exercise_collection_group(client, collection, patch_partition_queries):
     def _exercise_collection_group():
         collection_group = client.collection_group(collection.id)
         assert len(collection_group.get())
-        assert len([d for d in collection_group.stream()])
+        assert len(list(collection_group.stream()))
```

Comment:
Can we just use the thruthiness of lists here instead of calling len on all of these where we aren't checking for a specific length?
```suggestion
        assert list(collection_group.stream())
```

Reply:
Fixed with the pylint PR